abel/onion_bordas.py

```diff
@@ -109,7 +112,7 @@ def onion_bordas_transform(IM, dr=1, direction="inverse", shift_grid=False,
     # onion-peeling uses grid rather than pixel values, 
     # odd shaped whole images require shift image (-1/2, -1/2)
     if shift_grid:
-        IM = shift(IM, -1/2)
+        IM = shift(IM, -0.5, mode='nearest')
```

Comment:
`__future__.division` ensures `0.5`, roll on EOL python2.7. Personal preference for decimals, no worries. ;-)

Reply:
Yeah, living in the Py2 universe for the past 10 years has given me a deeply-rooted fear of integer division :)